recipes/openscienceplot_matplotlib/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "openscienceplot_matplotlib" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0beb2d70dcee50308123383353683fe4c483728cf85bacb29923e890160d2b81
+
+build:
+  number: 0
+  noarch: python
+  script_env: SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools_scm
+  run:
+    - python >=3
+    - numpy
+    - matplotlib
+    - h5py
+
+test:
+  imports:
+    - openscienceplot_matplotlib
```

Is there any reason for not including `pip check` here?
```suggestion
    - openscienceplot_matplotlib
  commands:
    - pip check
  requires:
    - pip
```

---

That I didn't know it existed! Thanks!